tests/integration/integration_steps/fabric_steps.py

```diff
@@ -0,0 +1,46 @@
+
+from azure.identity import DefaultAzureCredential
+from deltalake import DeltaTable
+from dateutil import tz
+import pandas as pd
+from pandas.testing import assert_frame_equal
+
+TIMESTAMP_COLUMN = "timestamp"
+
+def get_ts_delta_table(credential: DefaultAzureCredential, lakehouse_timeseries_path: str):
+    token = credential.get_token("https://storage.azure.com/.default")
+    return DeltaTable(lakehouse_timeseries_path,storage_options={"bearer_token": token.token, "use_fabric_endpoint": "true"},)
+
+def read_deltalake_timeseries(timeseries_path:str, credential: DefaultAzureCredential):
+    delta_table = get_ts_delta_table(credential, timeseries_path)
+    df = delta_table.to_pandas()
+    return df
+
+def prepare_lakehouse_dataframe_for_comparison(dataframe, external_id):
```

Comment:
some functions define return types, others don't. What's the criteria to determine if a return type is explicitly defined?

Reply:
We should probably align with Cognite, who seem to explicitly define return types.  Was an oversight on my part, I haven't done this on previous projects I worked on in Python